odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/mapper/EnumValueMapper.java

```diff
@@ -1,12 +1,26 @@
 package org.opendatadiscovery.oddplatform.mapper;
 
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+import org.mapstruct.NullValueCheckStrategy;
 import org.opendatadiscovery.oddplatform.api.contract.model.EnumValue;
 import org.opendatadiscovery.oddplatform.api.contract.model.EnumValueFormData;
 import org.opendatadiscovery.oddplatform.model.tables.pojos.EnumValuePojo;
 
+@Mapper(config = MapperConfig.class)
 public interface EnumValueMapper {
 
-    EnumValuePojo mapForm(final EnumValueFormData form, final Long datasetFieldId);
+    @Mapping(source = "datasetFieldId", target = "datasetFieldId")
+    @Mapping(target = "name", source = "form.name", qualifiedByName = "mapToTrimmedStringNamed")
+    @Mapping(target = "description", source = "form.description", qualifiedByName = "mapToTrimmedStringNamed",
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS)
```

Comment:
Old mapper had the null check, because description might be empty and calling `.trim()` on null would result in NPE
```java
        if (form.getDescription() != null) {
            pojo.setDescription(form.getDescription().trim());
        }
```
If you don't provide `nullValueCheckStrategy` to `@Mapping`, mapstruct generates something like this:
```
                    enumValuePojo.setDescription(this.mapToTrimmedString(form.getDescription()));
```
without any null checks for description.. Considering that `mapToTrimmedString` doesn't have null check as well, it will throw NPE. I've added `nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS` to be 100% with the logic of old mapper, but we can possibly move null check to  `mapToTrimmedString` and remove `nullValueCheckStrategy`. What do you think?

Reply:
oh, ok, thanks.
Both approaches are fine, I am ok with nullValueCheckStrategy